database/migrations/2024_03_23_073625_create_product_status_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('product_status', function (Blueprint $table) {
+            $table->id();
+            $table->string('status')->unique()->comment('1:Exhibit request -> 2:Waiting for valuation -> 3:Evaluation -> (7:Suspended ->) 4:Waiting for display(Coming Soon) -> 5:Exhibited -> 6:Sold out');
```

Comment:
use int for status

Reply:
Sorry this is a kind of master table , so this one must be not changable. I will modify this again.